extensions/wikia/MercuryApi/handlers/MercuryApiFilePageHandler.class.php

```diff
@@ -48,7 +48,15 @@ function ( $acc, $child ) {
 			return $item;
 		}, $fileUsageData['fileList']);
 
+		$anonRedir = F::app()->sendRequest(
```

Comment:
We might check if the call actually succeeds before fetching and using the data

Reply:
As far as I can tell this usage is standard and I can't see any place to check if it succeeded. I've added a check if $data['url'] is set.